R/model-summary.R

```diff
@@ -230,6 +230,8 @@ nonmem_summary <- function(
 
   res_list <- combine_list_objects(res_list, bbi_list)
 
+  res_list <- map_list_recursive(res_list, set_bbi_null)
+
```

Comment:
What do you think about moving this to `create_summary_object`?  That way gh-527 won't need to be adjusted to make a similar call.

Reply:
I looked into this earlier, but `model_summaries()` shouldn't need to be adjusted since it maps over `model_summary` (see below). Without doing too much of an investigation, it still seems like you're calling out to that in your PR. Am I missing something?
![image](https://user-images.githubusercontent.com/18685116/182633699-a5be0b99-539b-4e9a-b7d1-a0fa59a32a79.png)

Reply:
`model_summaries.list` branches based on the bbi version:

https://github.com/metrumresearchgroup/bbr/blob/ce785a40b9b5c5bc0d00a4fccd75dc227104f1ba/R/model-summaries.R#L178-L182

`model_summaries_concurrent` doesn't go through `model_summary`.

Reply:
Ahh, I didnt catch that. Ok yeah Ill nest that function inside `combine_list_objects` then. @seth127 pinging you so you're aware

Reply:
>  Ill nest that function inside combine_list_objects then

Why `combine_list_objects` (which is a general utility) rather than `create_summary_object`?

Reply:
Made this change in latest commit

Reply:
> > Ill nest that function inside combine_list_objects then
> 
> Why `combine_list_objects` (which is a general utility) rather than `create_summary_object`?

That was a typo my bad lol. Edited it but it likely showed the first version in the github email. Made the change to the correct function (`create_summary_object`)